CONTRIBUTING.md

```diff
@@ -112,6 +112,16 @@ $ cd tests/e2etest
 $ go test -v ./... -run TestE2E/../../examples/$EXAMPLE_NAME
 ```
 
+To update the expected `diff.patch` or `results.yaml`, run
```

Comment:
You can add, in what cases a contributor should invoke this.

Reply:
Done